ovmf-vars-generator

```diff
@@ -212,6 +212,8 @@ def test_keys(args):
 
 def parse_args():
     parser = argparse.ArgumentParser()
+    parser.add_argument('--arch', '-a', help='Architecture hint',
+                        choices=['x86_64', 'aarch64'])
```

Comment:
I suggest using the UEFI names `x64` & `aarch64`, since that's really what we're referring to here.

Reply:
Should we add default='x86_64' (or 'x64') here?